README.md

```diff
@@ -171,6 +172,24 @@ Passwordless.token_generator = -> (session) {
 
 Session is going to keep generating tokens until it finds one that hasn't been used yet. So be sure to use some kind of method where matches are unlikely.
 
+### Token and Session Expiry
+
```

Comment:
Maybe add a, very short, note on the difference between _timeout and expiration_ here?